RecordCollection/Views/Albums/Show.cshtml

```diff
@@ -0,0 +1,27 @@
+﻿@model AlbumViewModel
+
+<h1>Album Details</h1>
+
+<div>
+<h2>@Model.Album.Title</h2>
+<p>Artist: @Model.Album.Artist</p>
+<p>Release Year: @Model.Album.ReleaseDate</p>
+
+@if (Model.IsModernAlbum())
```

Comment:
You might want to consider keeping this logic (if/else statements) in the controller and bringing it to the view, maybe using ViewData. It can simplify things and can be better for the organization of code.